inc/shortcodes.php

```diff
@@ -270,34 +305,46 @@ function sell_media_checkout_shortcode() {
 
 			<div class="sell-media-checkout-button group">
 				<?php do_action( 'sell_media_above_checkout_button' ); ?>
-				<p><a href="javascript:void(0)" class="sell-media-cart-checkout sell-media-button"><?php _e( 'Checkout Now', 'sell_media' ); ?></a></p>
-				<p id="sell-media-continue-shopping">
-					<?php
-					$html  = __( 'or', 'sell_media' );
-					$html .= ' <a href="' . get_post_type_archive_link( 'sell_media_item' ) . '">';
-					$html .= __( 'continue shopping &raquo;', 'sell_media' );
-					$html .= '</a>';
-					echo apply_filters( 'sell_media_or_continue_shopping', $html );
-					?>
-				</p>
+				<p><a href="javascript:void(0)" class="sell-media-cart-checkout sell-media-button "><?php _e( 'Checkout Now', 'sell_media' ); ?></a></p>
+				<div class='wait-message' style="display:none;">
+					Please wait...
+				</div>
 				<?php
 				$settings = sell_media_get_plugin_options();
 				if ( ! empty( $settings->terms_and_conditions ) ) : ?>
 					<p id="sell-media-tos" class="text-center small quiet"><?php echo apply_filters( 'sell_media_tos_label', __( 'By clicking "Checkout Now", you are agreeing to our <a href="javascript:void(0);" class="sell-media-empty-dialog-trigger">terms of service</a>.', 'sell_media' ) ); ?></p>
 				<?php endif; ?>
 			</div><!-- .sell-media-checkout-button -->
 
+
 		</div><!-- .sell-media-totals -->
 
+		<?php do_action( 'sell_media_below_registration_form' ); ?>
+		<p id="sell-media-continue-shopping">
+	<?php
+	$html  = __( '', 'sell_media' );
```

Comment:
What is this?

Reply:
There was $html  = __( 'or' , 'sell_media' );  Now I have reverted it back